bundles/org.openhab.binding.powermax/src/main/resources/OH-INF/thing/serial.xml

```diff
@@ -7,6 +7,7 @@
 	<bridge-type id="serial">
 		<label>Serial Connection</label>
 		<description>This bridge represents the serial connection to the alarm system.</description>
+		<semantic-equipment-tag>NetworkAppliance</semantic-equipment-tag>
```

Comment:
It is not a network appliance, it is an alarm system, is there no tag appropriate for that ?

Reply:
see above

Reply:
For this one, AlarmSystem is expected.